recipe/meta.yaml

```
@@ -29,13 +30,10 @@ requirements:
     - setuptools
     - gsl
     - fftw
-    - blas
-    - mkl-devel {{ mkl }}  # [win and blas_impl == "mkl"]
     - dsdp
     - glpk
     # suitesparse is linked against on windows, and compiled-in on windows, because yeah, windows.
     - suitesparse          # [not win]
-
   run:
     - python
```

You can do the following to force MKL
```suggestion
    - python
    - libblas * *_mkl
```

---

I _don't_ want to force it - that's the point of this PR...